LB-Ferdi-Fernandez/leveret-integrator-test
# Submission for test

### Summary

Tier 1 Mid-Puzzle Animation for Shooting Range. Please see [docs](https://docs.google.com/document/d/1h9CkzNs1JrY9kVqmkUQYjzJ2zsfyIDy9q14It0aVJZs/edit?fbclid=IwAR0YXg-PxnRhsNKD7tItQWar6utrZ9ZCelMuN4B3Ds-UBXYo6GswvREKeB4).

### Notable Changes

- Warehouse environment setup
- Integrated characters and props.
- Setup unity timeline for MPA.
- Integrated sound effects.
- Integrated visual effects.

### Testing

1. Run unity.
2. Press `Start MPA` button.
3. Observe that the MPA cutscene plays correctly.
4. Wait until the cutscene finishes.
5. Press `Start MPA` button again.
6. Observe that the cutscene resets smoothly.

#### Others (Bonus)

1. Fixed unity chan shadow issue.
7. Added fade out effect on bullet holes.
